gtsfm/runner/run_scene_optimizer_colmaploader.py

```diff
@@ -13,37 +14,81 @@
 logger = logger_utils.get_logger()
 
 
+# fmt: off
+# Successive relaxation threshold pairs -- from strictest to loosest
+# `inlier ratio` is the minimum allowed inlier ratio w.r.t. the estimated model
+NUM_INLIERS_THRESHOLDS      =  [200, 175, 150, 125, 100, 75,  50,   25, 15]
+MIN_INLIER_RATIOS_THRESHOLDS = [0.6, 0.5, 0.4, 0.3, 0.2, 0.1, 0.1, 0.1, 0.1]
```

Comment:
What is the reasoning behind this? It does not make sense for me to use 15 inliers with a 0.1 ratio. From an estimation perspective I think inlier ratio is expected to be constant or increase as we decrease the number of inliers. 
Happy to look at some literature if this choice is based on something.

Reply:
@akshay-krishnan thanks for taking a look. 15 inliers with a 0.1 ratio is our current default. So I just relaxed it all the way back to where we are currently (which can be bad). But this makes a huge difference in performance.

I'll post the full metrics, but in the other PR here (which I will close), I found that for skydio-32, we can now get to:
- 6600 tracks after filtering after BA (from 612)
- 2.2 deg translation error after BA (from 9.0)
- 0.97 deg rotation error after BA (from 7.9)
- 0.12 translation distance error after BA (from 0.53)
- 2.91 avg track length unfiltered (from 2.76)
- 3.8 mean unfiltered reproj error (from 2829)

Reply:
Thanks @johnwlambert I think we should look beyond statistics here. It is not a deep model that estimates the F/E/H, and whatever hyperparameter should have some reasoning. 
I agree the current setting is not good, but as far as I can see setting a moderate number of inliers (like 50) with a high inlier ratio (0.6 or more) has not been tried in the other PR.

Reply:
And the fact that we are seeing so much improvement is some justification towards not using the 15, 0.1 setting we currently use.

Reply:
Basically I agree with most of your points, except that we should use inlier ratios as low as 0.1. As you noted, this may not give us a good model, so should be avoided.

Reply:
@akshay-krishnan I'm sure the thresholds can be tuned further. But I think this can help us get unblocked in terms of accuracy, and then we can try to tune these thresholds to work on many more datasets.

To put this in the two-view estimator, we are going to need to restructure the scene optimizer, two view estimator, and verifiers. I'd prefer to do that in a different PR so we could discuss the design in some more detail.

Reply:
Sure, it just becomes harder to do later as more PRs get merged. 
The thresholds can be tuned further, but why are we using a 0.1 inlier ratio if we know it doesn’t work well?